src/app/app.module.ts

```diff
@@ -12,10 +12,14 @@ import { FeatureModule } from '@app/feature/feature.module';
 import { CoreModule } from '@app/core/core.module';
 import { SharedModule } from '@app/shared/shared.module';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { FleetListComponent } from '@app/feature/fleet-list/fleet-list.component';
+import { NewFleetComponent } from '@app/shared/forms/new-fleet/new-fleet.component';
 
 const appRoutes: Routes = [
   { path: 'vehicles', component: VehiclesManagementComponent },
-  { path: 'add-vehicle', component: NewVehicleComponent }
+  { path: 'add-vehicle', component: NewVehicleComponent },
```

Comment:
pls ogarnij jeszcze jakieś sensowne urle typu vehicles, vehicles/add, fleets/add <- coś na takiej zasadzie moze? Potem będzie łatwiej to składać np. fleets/{id}/vehicles

Reply:
Poprawiono